system-int-test/src/cmr/system_int_test/data2/umm_spec_collection.clj

```diff
@@ -188,7 +189,12 @@
    :EntryTitle "The entry title V5"
    :DataDates [(umm-cmn/map->DateType {:Date (t/date-time 2012)
                                        :Type "CREATE"})]
-   :Abstract "A very abstract collection"})
+   :Abstract "A very abstract collection"
+   :MetadataSpecification (umm-c/map->MetadataSpecificationType
+                             {:URL (str "https://cdn.earthdata.nasa.gov/umm/collection/v"
+                                        umm-spec-versioning/current-collection-version),
+                              :Name "UMM-C"
+                              :Version umm-spec-versioning/current-collection-version})})
```

Comment:
It would be nice to put 
:MetadataSpecification (umm-c/map->MetadataSpecificationType
                             {:URL (str "https://cdn.earthdata.nasa.gov/umm/collection/v"
                                        umm-spec-versioning/current-collection-version),
                              :Name "UMM-C"
                              :Version umm-spec-versioning/current-collection-version})})
all into a function and just call that function for each of the 4 places where it gets used.

Reply:
Thanks for the suggestions. I think I am going to take care of this in my next ticket. I just want to merge this now that I finally got a green build.

Reply:
put a comment in the ticket so that we leave a trail